formats/src/main/scala/overflowdb/formats/ImporterMain.scala

```diff
@@ -34,7 +37,10 @@ object ImporterMain extends App {
             case Format.GraphMl => GraphMLImport
           }
           val odbConfig = overflowdb.Config.withoutOverflow.withStorageLocation(outputFile)
-          Using.resource(Graph.open(odbConfig, nodeFactories.asJava, edgeFactories.asJava)) { graph =>
+          val convertPropertyForPersistenceJava = new function.Function[Object, Object] {
+            override def apply(o: Object) = convertPropertyForPersistence(o).asInstanceOf[Object]
+          }
+          Using.resource(Graph.open(odbConfig, nodeFactories.asJava, edgeFactories.asJava, convertPropertyForPersistenceJava)) { graph =>
```

Comment:
Nit:

```
Using.resource(Graph.open(odbConfig, nodeFactories.asJava, edgeFactories.asJava, (o: Object) => convertPropertyForPersistence(o).asInstanceOf[Object]))```

also seems to work, i.e. no need to use `java.function.Function` explicitly

Reply:
true! even better IMO: `convertPropertyForPersistence(_).asInstanceOf[Object]`